seatunnel-transforms-v2/src/test/java/org/apache/seatunnel/transform/sql/zeta/ConcatWsFunctionTest.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.seatunnel.transform.sql.zeta;
+
+import org.apache.seatunnel.transform.sql.zeta.functions.StringFunction;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConcatWsFunctionTest {
+
+    @Test
+    public void testConcatWs() {
```

Comment:
Thanks @sohurdc ! Could you add some null value in array to test?

Reply:
ok